src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -0,0 +1,44 @@
+package mate.jdbc.model;
+
+public class Driver {
```

Comment:
create an empty constructor like in the Manufacturer model

Reply:
done